### PR TITLE
 BaSP-MR-162: Memberships Create Screen 

### PR DESCRIPTION
**Pull request by ale-schmidt**
You need to create a screen within the 'member' section that displays three available membership options along with their respective prices and descriptions. Additionally, please incorporate appealing styles to enhance the visual presentation.

- Create a new screen component within the 'member' section of the application.
- Retrieve the data for the three available membership options, including prices and descriptions.
- Display the membership options on the screen, showcasing their respective prices and descriptions.
- Apply appealing styles to enhance the visual presentation of the membership options.

### CODE REVIEW

**Review by marcohzo — APPROVED** [review]
Looks great, good job!

**Comment by ale-schmidt** [comment]
[[Member] Create memberships screen](https://trello.com/c/k97wqZLo/162-member-create-memberships-screen)

File: src/Components/Member/Memberships/index.jsx [hunk 1]
```diff
@@ -1,7 +1,99 @@
...
+              Only Clases
+            </button>
+            <button className={styles.button} onClick={() => setMemberType('classic')}>
+              Classic
+            </button>
+            <button className={styles.button} onClick={() => setMemberType('black')}>
+              Black
+            </button>
+            <button className={styles.button} onClick={() => setMemberType(false)}>
+              Sin uasurio
```
Comment (lorenzomauro59) [inline src/Components/Member/Memberships/index.jsx]:
```suggestion
              Sin usuario
```

**Review by lorenzomauro59 — APPROVED** [review]
look nice!! good job alee

**Review by fernandofernandezuy — APPROVED** [review]
Looks amazing

**Review by EmaBaldor — APPROVED** [review]
Great job!

**Comment by LucreciaV** [comment]
hi ale, I really like how it looks and when the memberships appear one by one, but I would center the part of the list on each membership and justify the text.

**Pull request merged**